Bazaro.Web/Program.cs

```diff
@@ -2,20 +2,34 @@
 using Bazaro.Data.Models;
 using Bazaro.Web.Areas.Identity;
 using Bazaro.Web.Data;
+using Bazaro.Web;
 using Blazorise;
 using Blazorise.Bootstrap5;
 using Blazorise.Icons.FontAwesome;
 using Microsoft.AspNetCore.Components;
 using Microsoft.AspNetCore.Components.Authorization;
 using Microsoft.EntityFrameworkCore;
 
+
```

Comment:
unnötige Leerzeile Entfernen

Reply:
Passt wird gemacht, da habe ich wohl nicht darauf geachtet in dem moment